<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmVoltageSource</class>
 <widget class="QMainWindow" name="frmVoltageSource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add voltage source</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Port1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Port2</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Voltage</string>
    </property>
    <widget class="QLabel" name="lblMagitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>236</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Magnitude(Vï¼‰</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>236</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Phase(degrees)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblImpedence">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>236</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference impedence(Ohm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMagnitude">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhase">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtImpedence">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtSourceName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
